2018-12-16 14:06:01,651 ERROR: Exception on /editrent/1481 [POST] [in c:\mjp\micro\env\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\mjp\micro\env\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\mjp\micro\env\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\MJP\micro\app\routes.py", line 48, in editrent
    thisprop = Property.query.filter(rent_id == id).first()
NameError: name 'rent_id' is not defined
2018-12-16 14:06:47,402 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 14:07:02,719 ERROR: Exception on /editrent/99 [POST] [in c:\mjp\micro\env\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\mjp\micro\env\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\mjp\micro\env\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\MJP\micro\app\routes.py", line 49, in editrent
    update_prop.propaddr = request.form["propaddr"]
NameError: name 'update_prop' is not defined
2018-12-16 14:10:42,779 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 17:28:00,684 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 18:51:57,880 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 18:52:27,310 ERROR: Exception on /editrent/1427 [POST] [in c:\mjp\micro\env\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\mjp\micro\env\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\mjp\micro\env\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\MJP\micro\app\routes.py", line 49, in editrent
    Print (thisprop)
NameError: name 'Print' is not defined
2018-12-16 18:53:07,601 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 18:53:28,224 ERROR: Exception on /editrent/1427 [POST] [in c:\mjp\micro\env\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\mjp\micro\env\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\mjp\micro\env\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\MJP\micro\app\routes.py", line 52, in editrent
    Print (thisagent)
NameError: name 'Print' is not defined
2018-12-16 18:54:19,492 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 19:00:36,117 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 19:00:50,721 ERROR: Exception on /editrent/1427 [GET] [in c:\mjp\micro\env\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\mjp\micro\env\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\mjp\micro\env\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\mjp\micro\env\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\MJP\micro\app\routes.py", line 70, in editrent
    print (rentdet.rent.source)
AttributeError: 'result' object has no attribute 'rent'
2018-12-16 19:05:47,273 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 19:45:36,113 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 19:50:59,383 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 22:10:29,748 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 22:10:41,756 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 22:10:53,548 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 22:11:05,396 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 22:28:01,014 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 22:28:10,345 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
2018-12-16 22:51:00,344 INFO: MicroJinn startup [in C:\MJP\micro\app\__init__.py:47]
